builders/build/substrate/customizing-chain-specs.md

```diff
@@ -0,0 +1,192 @@
+    "bootNodes": [
+        // boot nodes will be added automatically during deployment
+    ],
+    // Optional attribute, defaults to null
+    "telemetryEndpoints": null,
+    // Set a protocol identifier for this network
+    "protocolId": "container-chain-1000",
+    // Set properties to define the token and the ethereum compatibility
+    "properties": {
+        "isEthereum": true,
+        "ss58Format": 42,
+        "tokenDecimals": 18,
+        "tokenSymbol": "UNIT"
+    },
+    // Set the stagenet relay chain
+    "relay_chain": "westend_moonbase_relay_stagenet",
+    // Set the parachain id reserved in the relay chain
+    "para_id": 3333,
+    // No need
+    "codeSubstitutes": {},
+    "genesis": { 
+        ... 
+    }
+}
+```
+
+The other important section of the file is within the `genesis` attribute, which contains the genesis state. In the following JSON snippet, the default values and configuration for some modules are presented as an example: 
+
+```json
+{
+    ...
+    "genesis": {
+        "runtime": {
+            ...
+            // Sets the account that will bear sudo privileges
+            "sudo": {
+                "key": "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac"
+            },
+            // Sets the initial balances for some accounts
+            "balances": {
+                "balances": [
+                [
+                    "0x3cd0a705a2dc65e5b1e1205896baa2be8a07c6e0",
+                    1208925819614629174706176
+                ],
+                [
+                    "0x773539d4ac0e786233d90a233654ccee26a613d9",
+                    1208925819614629174706176
+                ],
+                [
+                    "0x798d4ba9baf0064ec19eb4f0a1a45785ae9d6dfc",
+                    1208925819614629174706176
+                ],
+                [
+                    "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac",
+                    1208925819614629174706176
+                ]
+                ]
+            },
+            // Sets balances for EVM accounts
+            "evm": {
+                "accounts": {
+                "0x1000000000000000000000000000000000000001": {
+                    "nonce": "0x1",
+                    "balance": "0xd3c21bcecceda1000000",
+                    "storage": {},
+                    "code": [
+                    0
+                    ]
+                },
+                "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b": {
+                    "nonce": "0x0",
+                    "balance": "0xffffffffffffffffffffffffffffffff",
+                    "storage": {},
+                    "code": []
+                },
+                "0xd43593c715fdd31c61141abd04a99fd6822c8558": {
+                    "nonce": "0x0",
+                    "balance": "0xffffffffffffffffffffffffffffffff",
+                    "storage": {},
+                    "code": []
+                }
+                }
+            },
+        }
+    }
+    ...
+}
+```
+
```

Comment:
A closing statement. It is always good to see something here, if not, it feels that the section just wraps without any meaning